presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/PartialAggregationPushDown.java

```diff
@@ -87,6 +87,12 @@ public PlanNode visitAggregation(AggregationNode node, RewriteContext<Void> cont
                 return context.defaultRewrite(node);
             }
 
+            boolean decomposable = node.isDecomposable(functionRegistry);
```

Comment:
I would just title this commit 'Factor decomposable check into separate method', with no message.